src/integrationTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/ReceivedMessagesHealthControllerTest.java

```diff
@@ -98,7 +98,7 @@ void test_health_reports_up_if_no_messages_received_in_last_hour_during_weekend(
     @Test
     void should_health_reports_up_when_no_messages_received_in_last_hour_during_holiday() throws Exception {
         // GIVEN
-        LocalDateTime localDateTime = LocalDateTime.of(2025, 8, 25, 12, 15);
+        LocalDateTime localDateTime = LocalDateTime.of(2025, 12, 25, 12, 15);
```

Comment:
How come you had to change this one?  Looks like it was already in 2025 so shouldn't be affected by the bank holidays update